internal/ethapi/api_test.go

```diff
@@ -96,6 +95,8 @@ func testTransactionMarshal(t *testing.T, tests []txData, config *params.ChainCo
 }
 
 func TestTransaction_RoundTripRpcJSON(t *testing.T) {
+	t.Parallel()
```

Comment:
Why is moving it to the outer better than the inner one?

Reply:
I think it’s easier to see that a test is running in parallel when `t.Parallel()` is placed at the outer level, as it clearly indicates the parallelism applies to the entire test case.